spec/features/admin_register_promotion_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+feature 'Promotion' do
+  let(:admin) do
+    create(:user, admin: true)
+  end
```

Comment:
👏 pequena sugestão pra ficar mais enxuto ainda

```suggestion
  let(:admin) { create(:user, admin: true) }
```